wordpress-mobile/gutenberg-mobile
# Fix for Enter.key pressed at the end of Heading block that does create another Heading block

This PR fixes the problem where pressing enter.KEY at the end of a Heading block, it does create another Heading block.  Fixes #734

Expectation: Pressing enter on a Heading block, should create a Paragraph block. Same as on any other "split-able" block.

**Testing A**
- Start the demo app
- Put the caret at the end of a Heading block
- Press enter.key
- It should create a new para after it

**Testing B**
- Start the demo app
- Put the caret at the middle of a Heading block
- Press enter.key
- It should create a new heading after it, with the content on the right of the caret

This PR is a bug fix, but we may need to put an eye on this other PR [here](https://github.com/WordPress/gutenberg/pull/10382) that modifies the default behaviour of the Heading block when pressing Enter.key at the beginning of it.
